.gitattributes

```diff
@@ -5,6 +5,9 @@
 *.vb linguist-vendored
 *.reg text
 
+# Collapse XLF files in PRs by default
+*.xlf linguist-generated=true
```

Comment:
Is there a way to expand them all through the UI?

Reply:
Yes.